### PR TITLE
chore(mesh): add navbar for 2.5

### PR DESCRIPTION
**Pull request by slonka**
### Description

Add navbar for 2.5 so it appears in the dropdown
 

### Testing instructions

Preview link: 

### Checklist 

- [X] Review label added 
- [X] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

**Comment by cloudjumpercat** [comment]
@fabianrbz Looks like the smoke test and netlify build failed for the same reason, do you know how we can fix it?

**Comment by fabianrbz** [comment]
for the tests, we should update [this line](https://github.com/Kong/docs.konghq.com/blob/main/.github/workflows/run-tests.yml#L13C27-L13C55) and default to something else in case it comes from a fork. Something like,

```
  VITE_PORTAL_API_URL: ${{ vars.VITE_PORTAL_API_URL || 'https://docs.konqhq.com' }}
```

I'll look into why it fails to build

**Comment by fabianrbz** [comment]
@cloudjumpercat @slonka we need to update two things:
* [this line](https://github.com/slonka/docs.konghq.com/blob/prepare-mesh-release-2.5/app/_data/docs_nav_mesh_2.5.x.yml#L6) should point to `../_src/.repos/kuma/app/_data/docs_nav_kuma_2.5.x.yml`
* we need to add the new version to [kong_versions.yml](https://github.com/slonka/docs.konghq.com/blob/prepare-mesh-release-2.5/app/_data/kong_versions.yml#L333-L339)

File: app/_data/docs_nav_mesh_2.5.x.yml [hunk 1]
```diff
@@ -0,0 +1,183 @@
+generate: true
+assume_generated: true
+max_depth: 3
+inherit:
+  path: /.repos/kuma/app/_src
+  nav: ../_src/.repos/kuma/app/_data/docs_nav_kuma_dev.yml
```
Comment (cloudjumpercat) [inline app/_data/docs_nav_mesh_2.5.x.yml:6]:
```suggestion
  nav: ../_src/.repos/kuma/app/_data/docs_nav_kuma_2.5.x.yml
```

**Review by cloudjumpercat — APPROVED** [review]
@slonka I'm going to approve this PR since the 2.5 docs render correctly in the preview even though one of the tests is failing. It sounds like we need to fix that test in a different PR. Most of the docs team will be out on the 13th, so I wanted to make sure this was approved so you can merge it in if the release goes out that day.

**Comment by cloudjumpercat** [comment]
@slonka @lahabana I think there will be merge conflicts between the two Mesh 2.5 PRs (this one and #6476 ), so I can drop in the morning of the 13th (CST) quickly to see if these can be merged or a docs member can help merge them when we're all back on Wednesday.

**Comment by lahabana** [comment]
Yeah this PR should have been a commit on: https://github.com/Kong/docs.konghq.com/pull/6476 just like this was done for Kuma-website

**Comment by slonka** [comment]
closing because of https://github.com/Kong/docs.konghq.com/pull/6495#issuecomment-1807964501 - will add there

**Comment by slonka** [comment]
@lahabana I thought that upgrade PR will just go through with auto-merge and then this PR would just work with "main"

**Pull request closed without merging**